src/central-intro.rst

```diff
@@ -17,28 +17,30 @@ Here are some of the major features we support today:
  - User accounts and management
  - Role-based user permissioning
  - Projects to organize users, permissions, and forms
- - Form and submission upload and management
+ - Form upload and management
 
    - With support for form version updates
    - With drafts and testing on initial creation, and on version updates
-   - With form and submission multimedia or data attachments
-   - With a table preview of submission data
+   - With form multimedia or data attachments
    - Encrypted forms (self-supplied or project managed keys)
+
+ - Submission upload and management
+
+   - From any compliant data collection client,
+   - Or directly from a web browser using `Enketo <https://enketo.org>`_ technology
+   - From permissions-managed known users or anonymous public links
+   - With submission multimedia or data attachments
+   - With an interactive table preview of submission data
    - OData live data feed for analysis with tools like Excel and Power BI
+   - Support for reviewing, commenting on, and editing submissions after upload
```

Comment:
I'd put the support line before the Odata one. And then change the OData one to
- Connect apps like Excel, Power BI, or R to create live-updating dashboards.